content/applications/productivity/spreadsheet/insert.rst

```diff
@@ -1,58 +1,821 @@
+  with the :ref:`appropriate formula <spreadsheet/insert/list-functions>` to retrieve the list
+  values. If there is corresponding data in your database, the cells are populated.
+
+  .. image:: insert/list-add-records.png
+     :alt: Add records by dragging the cell down
+
+- Position your cursor in the top left cell of the sheet, click :menuselection:`Data --> Re-insert
+  list` from the menu bar, then select the appropriate list. In the pop-up window, indicate the
+  number of records to insert and click :guilabel:`Confirm`. An updated list is inserted,
+  overwriting the previous list.
+
+.. tip::
+   The above methods can also be used to add additional blank rows to your spreadsheet table. This
+   may be useful for lists where you expect additional records to be generated in your database,
+   e.g., new product categories or new salespersons.
+
+.. _spreadsheet/insert/list-add-fields:
+
+Add fields/columns to a list
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To add fields/columns to a list:
+
+#. Select the column to the right or left of where the new column should be inserted.
+#. Click :menuselection:`Insert -->` :icon:`os-insert-col` :menuselection:`Insert column` then
+   :icon:`os-insert-col-before` :menuselection:`Column left` or :icon:`os-insert-col-after`
+   :menuselection:`Column right` from the menu bar, or right-click then :icon:`os-insert-col-before`
+   :guilabel:`Insert column left` or :icon:`os-insert-col-after` :guilabel:`Insert column right` as
+   appropriate.
+#. Copy the header cell of any column, paste it into the header cell of the new column, and press
+   `Enter`.
+#. Double-click the new header cell then click on the field name that appears in quotation marks at
+   the end of the formula; a list of all the technical names of the fields of the related model
+   appears.
+
+   .. image:: insert/list-add-columns.png
+      :alt: Add fields/columns by editing the formula
+
+#. Select the appropriate field name and press `Enter`. The field's label appears in the header.
+
+   .. tip::
+      To know a field's technical name, navigate to the relevant view, :ref:`activate developer mode
+      <developer-mode>`, then check the field name by hovering over the question mark beside a
+      field's label.
+
+#. With the header cell selected, double-click on the blue square in the bottom-right corner. The
+   cells of the column are populated with the appropriate formula to retrieve the list values. If
+   there is corresponding data in your database, the cells are populated.
+
+.. _spreadsheet/insert/list-duplicate:
+
+Duplicate a list
+~~~~~~~~~~~~~~~~
+
+Duplicating a list via the list's properties creates an additional data source. This allows for
+different manipulations to be performed on the same data within one spreadsheet.
+
+With the :ref:`list properties <spreadsheet/insert/list-properties>` open, click the :icon:`fa-cog`
+:guilabel:`(gear)` icon then :icon:`fa-clipboard` :guilabel:`Duplicate`.
```

Comment:
```suggestion
:guilabel:`(gear)` icon then :icon:`fa-clone` :guilabel:`Duplicate`.
```
fa-clone is the duplicate icon :)
![image](https://github.com/user-attachments/assets/9a901f8e-f4d9-41e0-b706-555a712356e0)